plugins/modules/ise_radius_integration_workflow_manager.py

```diff
@@ -736,6 +736,47 @@ def get_have(self, config):
         self.status = "success"
         return self
 
+    def get_primary_ise_fqdn(self, cisco_ise_dtos, ip_address):
+        """
+        Get the fully qualified domain name (fqdn) from the Multi-Node ISE environment
+        in the Cisco Catalyst Center with the role as 'primary'.
+
+        Parameters:
+            cisco_ise_dtos (list of dict): List of configuration of Multi-Node in an
+            ISE environment.
+            ip_address (str): IP address of the Cisco ISE server.
+
+        Returns:
+            primary_fqdn (str): The fqdn value of the primary node of the ISE server.
+        """
+
+        self.log("Getting the fully qualified name of the primary node of the ISE server...")
+        primary_node_ise_details = get_dict_result(cisco_ise_dtos, "role", "PRIMARY")
+        if not primary_node_ise_details:
+            self.msg = (
+                "There is no details of the PRIMARY role node in the multi-node ISE environment {ip_address}"
```

Comment:
"There are no details of the 'PRIMARY' role node in the multi-node ISE environment {ip_address}."
(OR)  
"No details found for the 'PRIMARY' role node in the multi-node ISE environment {ip_address}."